src/components/forms/ChallengeDetailsForm.tsx

```diff
@@ -84,11 +93,43 @@ const ChallengeDetailsForm = (props: any) => {
 
             {errors.name && <Text>This is required.</Text>}
 
+            {/* TODO: change to carousel */}
+            <Stack flexDirection="row" gap={18} pt={22} justifyContent="center">
+              <TouchableOpacity onPress={() => setIcon("icon1")}>
+                <Image
+                  source={require("../../../assets/img/mascotCreateChallenge1.png")}
+                />
+              </TouchableOpacity>
+              <TouchableOpacity onPress={() => setIcon("icon2")}>
+                <Image
+                  source={require("../../../assets/img/mascotCreateChallenge2.png")}
+                />
+              </TouchableOpacity>
+              <TouchableOpacity onPress={() => setIcon("icon3")}>
+                <Image
+                  source={require("../../../assets/img/mascotCreateChallenge3.png")}
```

Comment:
I think it would be better to use the SVG files if possible. At the moment we only have the Icon SVG component, I created on for badges. Maybe we can create one for Image SVGs.

Reply:
I agree, thanks Rein. This PR was from the spillover ticket last sprint, which is focused on flow and API integration. So there are no styling yet. 🙏